internal/schemautil/service.go

```diff
@@ -636,12 +636,17 @@ func copyServicePropertiesFromAPIResponseToTerraform(
 		}
 	}
 
-	userConfig, err := apiconvert.FromAPI(userconfig.ServiceTypes, serviceType, s.UserConfig)
+	newUserConfig, err := apiconvert.FromAPI(userconfig.ServiceTypes, serviceType, s.UserConfig)
```

Comment:
what would be the need to rename this variable?

Reply:
To keep tf style old vs new.

Reply:
please see https://github.com/aiven/terraform-provider-aiven/pull/1000/files#r1048277751